Uppsala/UppEt58.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="UppEt58">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title xml:id="t1">
+                    Magic amulet
+                </title>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0. 
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0356Upp"/>
+                        <collection>O Etiop.</collection>
+                        <idno>O Etiop. 58</idno>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        
+                        <msItem xml:id="ms_i1">
+                            <locus target="#1r"/>
+                            <title type="incomplete" ref="LIT2842RepCh125"/>
+                            <incipit xml:lang="gez">
+                                ሰላም ፡ ለገቦከ ፡ ኲናተ ፡ ለንጊኖስ ፡ ዘወግዖ ። ወ<surplus>አ</surplus>አዘቅተ ፡ ማይ ፡ 
+                                ከርሥከ ፡ ለደመ ፡ ሥርአት<surplus>ት</surplus>፡ እንተ ፡ አንቅዖ ፡ ሐዋርያ ፡ አብ ፡ 
+                                ክርስቶስ ፡ ዘፈጸምከ ፡ ግብረ ፡ ተሰብኦ ፡ ዕቀበኒ ፡ በመስቀልከ ፡ 
+                                ለመልአከ ፡ዓለም ፡ ዘሞዖ ፡ ከመ ፡ኖላዊ ፡ የዓቅብ፡ 
+                                እምተኵላ ፡ በግዖ ፡ ኢጢሚሶር ፡ ኢጢሚሶር ፡ ኢጢሚሶር ፡ 
+                                ኢጤሜሶር ፡ ኢጢሚሶር ፡ ኤጤሜሶር ፡ ኤጤሜሶር ፡ 
+                                ግድምናሶር ፡ ግድምናሶር ፡ ግድምናሶር ፡ ግድምናሶር ፡ 
+                                ግድምናሶር ፡ ግድምናሶር ፡ ልድሞናሶር ፡ ፈልዳማአስ ፡ 
+                                ፈልዳማአስ ፡ ፈልዳማአስ ፡ ፈልዳማአስ ፡ ፈልዳማአስ ፡ 
+                                ፈልዳማአስ ፡ ፈልዳማአስ ፡ ግቶሞር ፡ ግቶሞር ፡ግቶሞር ፡
+                                ግቶሞር ፡ ግቶሞር፡ ግቶሞርግቶሞር። <add place="above">በኃይለ፡</add> እሉ ፡ አስማቲከ፡ 
```

Comment:
Is it ok to have sometimes a space before ፡ and sometimes not?

Reply:
The spaces are chaos. Thanks for noticing it, @nafisa-valieva and @MassimoVilla !